src/main/quickSetStrip/qssDialog.js

```diff
@@ -0,0 +1,144 @@
+
+    components: {
+        channelNotifier: {
+            type: "gpii.app.channelNotifier",
+            options: {
+                events: {
+                    onQssOpen: "{qss}.events.onQssOpen",
+                    onQssWidgetToggled: "{qss}.events.onQssWidgetToggled",
+                    onSettingUpdated: "{qss}.events.onSettingUpdated"
+                },
+                listeners: {
+                    onSettingUpdated: {
+                        "funcName": "console.log",
+                        args: ["Sending Updated QSS: ", "{arguments}.0"]
+                    }
+                }
+            }
+        },
+        channelListener: {
+            type: "gpii.app.channelListener",
+            options: {
+                events: {
+                    onQssClosed: null,
+                    onQssButtonFocused: null,
+                    onQssButtonsFocusLost: null,
+                    onQssButtonActivated: null,
+                    onQssButtonMouseEnter: null,
+                    onQssButtonMouseLeave: null,
+
+                    onQssSettingAltered: "{qss}.events.onQssSettingAltered"
+                },
+
+                listeners: {
+                    onQssClosed: {
+                        func: "{qss}.hide"
+                    },
+                    // XXX DEV
+                    onQssButtonActivated: {
+                        funcName: "console.log",
+                        args: ["Item clicked: ", "{arguments}.0"]
+                    },
+                    // onQssButtonMouseEnter: {
+                    //     funcName: "console.log",
+                    //     args: ["Item Enter: ", "{arguments}.0.target.offsetLeft"]
+                    // },
+                    // onQssButtonMouseLeave: {
+                    //     funcName: "console.log",
+                    //     args: ["Item Leave: ", "{arguments}.0.target.offsetLeft"]
+                    // },
+                    onQssSettingAltered: {
+                        funcName: "console.log",
+                        args: ["Setting altered QSS:", "{arguments}.0.path", "{arguments}.0.value"]
+                    }
+                }
+            }
+        }
+    },
+    listeners: {
+        "onCreate.initBlurrable": {
+            func: "{that}.initBlurrable",
+            args: ["{that}.dialog"]
+        }
+    },
+    invokers: {
+        show: {
+            funcName: "gpii.app.qss.show",
+            args: [
+                "{that}",
+                "{arguments}.0" // params
+            ]
+        }
+    }
+});
+
+/**
+ * Show the window in case QSS is not disabled.
+ */
+gpii.app.qss.show = function (that, params) {
+    if (!that.options.disabled) {
+        // Show the QSS or focus it if it is already shown.
+        if (that.model.isShown) {
+            that.focus();
+        } else {
+            that.applier.change("isShown", true);
+        }
+
+        that.events.onQssOpen.fire(params);
```

Comment:
we could remove this and simply use the `dialog.events.onDialogShown` event triggered in dialog - `toggle` function (connected with isShown property)